### DISCUSSION TITLE
When closing a workspace, switch to the last active or any other active workspace.

### DISCUSSION THREAD

**Discussion by haydenjames** [i3/i3#5726] [Q&A]
If I'm not mistaken I had this working previously on another laptop. Lost the dotfiles accidentally when pruning cloud backup. 

When closing a workspace can I see i3wm to auto-switch to the last active or any other active workspace?

**Comment by orestisfl** [comment]
i3 does not have the concept of "closing" a workspace, a workspace is deleted when it's not visible and there is no window on that workspace.

If you mean to auto-switch to the last active workspace when a workspace becomes empty you can certainly do that with an [IPC script](https://i3wm.org/docs/ipc.html).

**Comment by haydenjames** [comment]
Thanks for the background clarification and for the link. I'm only passing through, so I will not be up to speed on many of the i3wm concepts and terminology. Wish I had the time. 

I believe I still have my previous solution working on an old Pinebook Pro. I am traveling now,  but when I return, once it charges and if I find it, I will share a link and/or exact details on the solution for anyone else also searching.

Thanks for the quick reply.
